README.build.md

```diff
@@ -19,9 +19,38 @@ export SDE_INSTALL=<path>
 ## Build
 
 ```bash
-./make-all.sh
+./make-all.sh --target tofino
 ```
 
-The `make-all` script creates a separate install tree for OVS.
+## General options
 
-This allows you to rebuild the non-OVS code without having to rebuild OVS.
+| Parameter | Value | Description |
+| --------- | ----- | ----------- |
+| `--clean` | | Remove the main build and install trees. |
+| `--prefix` |  _path_ | Path to the directory in which build artifacts should be installed. Sets the  `CMAKE_INSTALL_PREFIX` CMake variable. Default value is `./install`. |
+| `-sde-install` | _path_ | Path to install directory for the target driver (SDE). Sets the `SDE_INSTALL_PREFIX` CMake variable. Defaults to the value of the `SDE_INSTALL` environment variable. |
```

Comment:
--sde-install? One hyphen is missing

Reply:
I submit that it was a minus sign, not a hyphen, because it clearly subtracted one character.

In fact, it must have been a predecrement operator, because it took it off the beginning of the string, before you even looked at it.

_badoom-crash..._